lago/vm.py

```diff
@@ -123,6 +127,10 @@ def bootstrap(self):
                         ),
                         sysprep.set_iscsi_initiator_name(self.vm.iscsi_name()),
                         sysprep.set_selinux_mode('enforcing'),
+                        sysprep.edit(
+                            "/boot/grub2/grub.cfg",
+                            "s/set timeout=5/set timeout=0/g"
+                        ) if self.vm.distro() == 'el7' else '',
```

Comment:
is there a reason to restrict it to el7 only? (at least on my laptop with fc24 it looks like the same substitution would work)